radixdlt-java/radixdlt-cli/src/main/java/com/radixdlt/cloud/AWSSecrets.java

```diff
@@ -0,0 +1,123 @@
+
+import com.radixdlt.cli.OutputCapture;
+import com.radixdlt.cli.RadixCLI;
+import com.radixdlt.utils.AWSSecretManager;
+import com.radixdlt.utils.AWSSecretsOutputOptions;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+public class AWSSecrets {
+
+	private static final Boolean DEFAULT_ENABLE_AWS_SECRETS = false;
+	private static final Boolean DEFAULT_RECREATE_AWS_SECRETS = false;
+	private static final String DEFAULT_NETWORK_NAME = "testnet";
+
+	private AWSSecrets() {
+	}
+
+	public static void main(String[] args) {
+
+		Options options = new Options();
+		options.addOption("h", "help", false, "Show usage information (this message)");
+		options.addOption("n", "full-node-number", true, "Number of full nodes");
+		options.addOption("as", "enable-aws-secrets", true, "Store as AWS Secrets(default: " + DEFAULT_ENABLE_AWS_SECRETS + ")");
+		options.addOption("rs", "recreate-aws-secrets", true, "Recreate AWS Secrets(default: " + DEFAULT_RECREATE_AWS_SECRETS + ")");
+		options.addOption("k", "network-name", true, "Network name(default: " + DEFAULT_NETWORK_NAME + ")");
+
+		CommandLineParser parser = new DefaultParser();
+		try {
+			CommandLine cmd = parser.parse(options, args);
+			if (!cmd.getArgList().isEmpty()) {
+				System.err.println("Extra arguments: " + cmd.getArgList().stream().collect(Collectors.joining(" ")));
+				usage(options);
+				return;
+			}
+
+			if (cmd.hasOption('h')) {
+				usage(options);
+				return;
+			}
+
+			final int fullNodeCount = Integer
+				.parseInt(getOption(cmd, 'n').orElseThrow(() -> new IllegalArgumentException("Must specify number of full nodes")));
+			if (fullNodeCount <= 0) {
+				throw new IllegalArgumentException("There must be at least one full node");
+			}
+
+			final String networkName = getOption(cmd, 'k').orElse(DEFAULT_NETWORK_NAME);
+			final boolean enableAwsSecrets = Boolean.parseBoolean(cmd.getOptionValue("as"));
+			final boolean recreateAwsSecrets = Boolean.parseBoolean(cmd.getOptionValue("rs"));
+
+			final AWSSecretsOutputOptions awsSecretsOutputOptions = new AWSSecretsOutputOptions(enableAwsSecrets, recreateAwsSecrets, networkName);
+			IntStream.range(0, fullNodeCount).forEach(i -> {
+				final String keyStoreName = String.format("fullnode%s.ks", i);
+				final String passwordName = String.format("fullnode%s-password", i);
+				final String keyFileSecretName = String.format("%s/%s", networkName, keyStoreName);
+				final String passwordSecretName = String.format("%s/%s", networkName, passwordName);
```

Comment:
same as above

Reply:
Password is testnet/fullnode1/fullnode1-password